### PR TITLE
Add SBLYR Pool to registry.json

### PR DESCRIPTION
**Pull request by keyflight**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -1787,5 +1787,20 @@
...
+    "poolId": "8264de3cdb1798dd8758e24cda5101184b44543e7c4421c7815f9ed8",
+    "social": {
+      "twitter": "sublayerio",
+      "facebook": "",
+      "reddit": "",
+      "instagram": "",
+      "website": "https://pool.sublayer.io",
+      "youtube": ""
+    },
+    "telegram": "",
```
Comment (lophre) [inline registry.json:1802]: You must provide a Telegram username for SPA voting purposes.

File: registry.json [hunk 2]
```diff
@@ -1787,5 +1787,20 @@
...
+    "operatorName": "PiTrippy",
```
Comment (lophre) [inline registry.json:1803]: If you don't provide a real first name or full name, please leave this field empty.

**Review by lophre — CHANGES_REQUESTED** [review]
Please see comments

**Comment by keyflight** [comment]
@lophre Thanks! I've created a new PR with the right changes. https://github.com/SinglePoolAlliance/Registration/pull/224

**Pull request closed without merging**